src/components/Product/ProductList/ProductList.jsx

```diff
@@ -60,11 +62,14 @@ export default function ProductList(props) {
             {isModalActive && (
                 <Modal
                     imageValue={imageValue}
-                    setIsModalActive={setIsModalActive}
+                    isOpen={() => setIsModalActive(true)}
+                    onClose={() => setIsModalActive(false)}
+                    portalClassName={"body"}
+                    classNameWrapper={"product_list_row"}
```

Comment:
класи пропсами передавай просто як строку: 
```suggestion
                    classNameWrapper="product_list_row"
```

Поправ так всюди